src/test/cypress/e2e/integration/app-frontend/redirect.js

```diff
@@ -18,7 +18,7 @@ describe('Redirect', () => {
     cy.get(appFrontend.altinnError).should('contain.text', texts.missingRights);
   });
 
-  it('Unknown error page', () => {
+  it('User is redirected to unknown error page when a network call fails', () => {
```

Comment:
closer...
try to formulate the sentences like they start with it:
`it('redirects user to unknown error page when a network call faills',() => {`

Reply:
I agree that formulating the test names with `it` gives better code readability, but the test results are published to the build pipeline like [this](https://dev.azure.com/brreg/altinn-studio/_build/results?buildId=195531&view=ms.vss-test-web.build-test-results-tab) without `it` and the same applies to the test name displayed in cypress dashboard.